packages/editor/src/components/provider/index.native.js

```diff
@@ -69,6 +69,10 @@ class NativeEditorProvider extends Component {
 	}
 
 	componentDidMount() {
+		const { capabilities } = this.props;
+
+		this.props.updateSettings( capabilities );
+
```

Comment:
What does this change do?

Reply:
This sets the initial capabilities given by initial props. In this way we avoid a re-rendering.

Reply:
This change looks to be just reordering the lines, accessing `capabilities` separately and calling `updateSettings` before `subscribeParentGetHtml`.

It's still not clear to me how this avoid a re-rendering, but happy to leave it as-is.